finclab/my-wealth-frontend
# Title: set up react base app for the application

#### What does this PR do?
- This PR implement the base app configuration for the react application

#### Description of Task to be completed?
- Set base configuration for the react application.
- Set up react server
- Implement react to run on port 3000

#### How should this be manually tested?
- clone repo by running `npm clone github repository url`
- Cd into the directory and run `npm install` to install all dependencies
- Visit `http://localhost:3000/` to see the react application on the browser

#### Any background context you want to provide?
- None

#### Screenshots (if appropriate)
<img width="896" alt="Screen Shot 2019-04-08 at 6 35 57 AM" src="https://user-images.githubusercontent.com/32917511/55700808-a3ec1480-59c8-11e9-9180-946612a9c21b.png">

#### What are the relevant pivotal tracker stories?
[#165172328](https://www.pivotaltracker.com/story/show/165172328)

#### Questions:
- None
